docker-compose.yaml

```diff
@@ -0,0 +1,18 @@
+version: "3.8"
+
+services:
+    nats:
+        image: nats:latest
+        command: []
+        ports:
+            - "4222:4222"
+            - "6222:6222"
+            - "8222:8222"
+    authorization-service:
+        image: jwalab/authorization-service:latest
```

Comment:
let's pin this version, the latest one for auth service is 0.0.2

Reply:
Updated. Thank you.